src/containers/Review/overview/Overview.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { DateTime } from 'luxon'
+import { Icon, Message, Segment } from 'semantic-ui-react'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import useLocalisedEnums from '../../../utils/hooks/useLocalisedEnums'
+import { FullStructure } from '../../../utils/types'
+import { ActivityLog, ApplicationOutcome } from '../../../utils/generated/graphql'
+
+export const Overview: React.FC<{
+  structure: FullStructure
+  activityLog: ActivityLog[]
+}> = ({
+  structure: {
+    info: { current, outcome, user, org, serial },
+  },
+  activityLog,
+}) => {
+  const { Outcome } = useLocalisedEnums()
+  const applicant = `${user?.firstName} ${user?.lastName}`
+  const organisation = org?.name
+  const { started, completed } = getDates(activityLog)
+  const stage = current.stage.name
+
+  return (
+    <div id="overview">
+      <Segment basic>
+        <Message info icon>
+          <Icon name="info circle" color="teal" />
+          <Message.Content>
+            <div className="flex-row wrap">
+              <p className="left-item">
+                <strong>Applicant: </strong>
+                {applicant}
+              </p>
+              <p className="right-item">
+                <strong>Organisation: </strong>
+                {organisation}
+              </p>
+            </div>
+            <div className="flex-row wrap">
+              <p className="left-item">
+                <strong>Started: </strong>
+                {DateTime.fromISO(started).toLocaleString()}
+              </p>
+              <p className="right-item">
+                <strong>Completed: </strong>
+                {completed && DateTime.fromISO(completed).toLocaleString()}
+              </p>
+            </div>
+            <div className="flex-row wrap">
+              <p className="left-item">
+                <strong>Stage: </strong>
+                {stage}
+              </p>
+              <p className="right-item">
+                <strong>Outcome: </strong>
+                {Outcome[outcome as ApplicationOutcome]}
+              </p>
+            </div>
+            <div className="flex-row wrap">
+              <p className="left-item">
+                <strong>Serial: </strong>
+                {serial}
+              </p>
```

Comment:
All localization missing. Sorry to be picking on this, but it would be good to already add there as well.